trigger/task.go

```diff
@@ -259,13 +259,15 @@ func (t *taskTriggers) makeData(sub *event.Subscription, pastTenseOverride, test
 
 	displayName := t.task.DisplayName
 	status := t.task.Status
-	parentDisplayTask, err := t.task.GetDisplayTask()
-	if err != nil {
-		return nil, errors.Wrap(err, "error getting display task")
-	}
-	if parentDisplayTask != nil {
-		displayName = t.task.DisplayTask.DisplayName
-		status = t.task.DisplayTask.Status
+	if !useExecTaskStatus {
```

Comment:
are there any task triggers we still do this for?

Reply:
The ones that alert on the display task do this

Reply:
but in those cases they never have a parent display task.

Reply:
ah I missed that. Removed this entire block